Automattic/wp-calypso
# Color Schemes: Remove duplicated Thumbnails from public/

#### Changes proposed in this Pull Request

Color Schemes: Remove duplicated Thumbnails from public/

`public/` is now disposable. Our static assets live in `static/` now, and are copied to `public/` as part of the build process.

It seems like these two SVGs accidentally slipped in with #32794 and #33136, respectively. The same two files were also added to the correct location in `static/`, so we don't have to move them there but can simply drop them. 

#### Testing instructions

Verify that the app still builds, and color schemes still work; verify especially that the thumbnails for the 'Sunset' and 'Contrast' schemes are present.
